maas-schemas/schemas/core/components/customerSelectionV2.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$id": "http://maasglobal.com/core/components/customerSelectionV2.json",
+  "description": "Customer Selection for Configurator v2",
+  "type": "object",
+  "patternProperties": {
+    "^": {
+      "$ref": "http://maasglobal.com/core/components/customerSelectionCommon.json#/definitions/configChoiceSelection"
+    }
```

Comment:
Does pattern properties work actually? The last time I used it at least the ts convert script is throwing warning 🤔